src/Client/components/ui/Modal/LoadingModal.tsx

```
@@ -0,0 +1,25 @@
+  isOpen: boolean;
+  message?: string;
+}
+
+const LoadingModal: React.FC<LoadingModalProps> = ({ isOpen, message = 'Loading...' }) => {
+  if (!isOpen) {
+    return null;
+  }
+  return (
+    <div className={styles.modalOverlay}>
+      <div className={`${styles.modalContent} dark:bg-[#18181b] dark:text-white`}>
+        <div className={styles.spinner}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)